tests/cli/test_formatters.py

```diff
@@ -1357,27 +1357,34 @@ def test_tpu_container(self) -> None:
 
 
 class TestConfigFormatter:
-    async def test_output(self, root: Root) -> None:
+    @pytest.mark.parametrize("platform", ["linux", "win32"])
```

Comment:
Don't use parametrization here.
We run tests on both Linux and Windows.
No need to monkey-patch platform for OS simulation if we have a test against this OS already.